### DISCUSSION TITLE
AspNetCore SDK Ideas

### DISCUSSION THREAD

**Discussion by facus26** [scalar/scalar#3006] [Ideas]
Hi everyone,

I have a fork of the project with some changes for the .NET SDK. 
1. I added support for some missing options. 
2. I also added documentation for the options. 
3. I Created an enum for the themes and a static class with the strings for Clients.

**Comment by hanspagel** [comment]
This one, right? https://github.com/facus26/scalar

Nice! There’s a big PR that was opened just now: #3019

Maybe you can submit a few PRs once that’s in, too? 😊

**Comment by xC0dex** [comment]
Thanks again for the contribution and inspiration to contribute to this awesome project. 🥳 I'll close this discussion!
